### PR TITLE
Tribute Page

### PR DESCRIPTION
**Pull request by banzaman**
- Added Linters.
- Added HTML file.
- Added CSS file.
- Added README file.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="css/styles.css"
+      media="screen"
+    />
+  </head>
+  <body>
+    <article>
+      <!-- title -->
```
Comment (banzaman) [inline index.html:19]: Please remove commented code from the whole file as it can make the whole codebase unclean.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (banzaman) [inline README.md:3]: please add a description of your project in the `README` file, recruiters will be grateful to have a preview of the project 🚀

File: README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (banzaman) [inline README.md:39]: - [ ] [OPTIONAL] Please, consider to fix your MIT file correctly displaying. It has the link, but you do not have a MIT.md file in your repo.

**Review by banzaman — COMMENTED** [review]
Hi @banzaman, 

Great job so far making your first review 🙌!
There is one more issue that needs to be addressed to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by gemmen29 — APPROVED** [review]
Hi @banzaman, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**